.github/workflows/build_and_publish.yaml

```diff
@@ -29,6 +29,5 @@ jobs:
           ssh-add - <<< "${{ secrets.DOKKU2_DEPLOY_SSH_KEY }}"
     - name: Deploy
       run: |
-          IMAGE="ghcr.io/opensafely-core/$IMAGE_NAME:latest"
-          ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" dokku@dokku2.ebmdatalab.net "docker pull $IMAGE && docker tag $IMAGE dokku/proxy && dokku tags:deploy proxy"
+          ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" dokku@dokku2.ebmdatalab.net git:from-image proxy $IMAGE_ID
```

Comment:
should this be 

```
ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" dokku@dokku2.ebmdatalab.net dokku git:from-image proxy $IMAGE_ID
```

?

Reply:
No, the SSH config for sshing into automatically runs dokku and passes your dokku args

That's why the previous appraoch failed, it couldn't run the `docker pull` command. See: https://github.com/opensafely-core/proxy/runs/3590554847?check_suite_focus=true